src/workspace/config/snapshot.json

```diff
@@ -327,7 +327,7 @@
                                 },
                                 "webAPIs": ["notifications", "audio", "video"]
                               },
-                              "url": "<RT_URL>/tiles",
+                              "url": "<RT_URL>/fx-fx-tiles",
```

Comment:
Looks like a mistake `fx-fx`

Reply:
yes, damn!

Reply:
weird that it worked (I did test it !!) .. that section must not be active

Reply:
fixed now